src/protocols/ethernet/MDI/MDI-1000Base-T.stanza

```diff
@@ -71,6 +71,9 @@ are expected to be passed in the right order - otherwise,
 we may setup the wrong pin assignment constraints.
 <DOC>
 public defn make-1000Base-T-supports (dps:JITXObject ...):
+   make-1000Base-T-supports(dps)
+
+public defn make-1000Base-T-supports (dps:Tuple<JITXObject>):
```

Comment:
Why do we need this ?  In absence of this function, you should be able to do:

```
make-1000Base-T-supports(dps = [...])
```

In other words - when you use a `variadic` like `dps` above, you can also pass it as a named keyword argument.

Reply:
I think it's helpful to have a convenience function to handle the case where the user passes in a tuple without the named keyword argument. Otherwise they'll get 

```
Cannot call function 'make-1000Base-T-supports' with given arguments:
  Argument 0: Passing type 'Tuple<JITXObject>' to expected type 'JITXObject'.
stanza> 
```

Reply:
OK :+1: